### DISCUSSION TITLE
Screenshots not attached to allure rports

### DISCUSSION THREAD

**Discussion by sleepstream** [selenide/selenide#2582] [Q&A]
Hi!

When I start tests using maven I have problem with attaching screenshots from failed tests to allure reports.

All screenshots have been made and exist in Configuration.reportsFolder but have not been attached in allure-results.

But if I start tests manually from IDE - all works well.

```
<dependency>
            <groupId>com.codeborne</groupId>
            <artifactId>selenide</artifactId>
            <version>6.17.1</version>
        </dependency>
        <dependency>
            <groupId>com.codeborne</groupId>
            <artifactId>selenide-core</artifactId>
            <version>6.17.1</version>
        </dependency>
        <dependency>
            <groupId>com.codeborne</groupId>
            <artifactId>selenide-proxy</artifactId>
            <version>6.17.1</version>
        </dependency>
        <dependency>
            <groupId>com.codeborne</groupId>
            <artifactId>selenide-testng</artifactId>
            <version>6.16.0</version>
        </dependency>
```

**Comment by dulciscrustulam** [comment]
You did not attach any info regarding your implementation, but you may try the following, in case you did not do that. Otherwise it is better to provide more info in your question.

`SelenideLogger.addListener("AllureSelenide",  new AllureSelenide().screenshots(true).savePageSource(false));`
